utils/time.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Converts Time in milliseconds to Epoch Time
```

Comment:
Shouldn't this be `seconds`?

Reply:
We are converting from milliseconds to Epoch (seconds).
`Date.now( )` method returns the number of milliseconds elapsed since January 1, 1970

Reply:
Got it

Reply:
Slight clarification here:
Epoch is a particular point in time: 00:00:00:0000 on Jan 1, 1970
Unix timestamp (or Unix time or POSIX time) is the number of seconds since epoch.